torch/csrc/jit/passes/mkldnn_rewrite.cpp

```diff
@@ -13,6 +13,141 @@ namespace jit {
 
 #if AT_MKLDNN_ENABLED()
 
+namespace mkldnn {
+
+#define ATTR_MAP_ITEM(NAME)                                           \
+  {                                                                   \
+#NAME, {                                                          \
+      [](std::vector<c10::optional<at::Scalar>> scalars,              \
+         c10::optional<std::string> algorithm) {                      \
+        return ideep::attr_t::fuse_##NAME();                          \
+      },                                                              \
+          zero_scalar_operand, op_list_construct(zero_scalar_operand) \
+    }                                                                 \
+  }
+
+static constexpr float kMin = -std::numeric_limits<float>::infinity();
+static constexpr float kMax = std::numeric_limits<float>::infinity();
+
+const std::vector<std::string> zero_scalar_operand =
+    std::vector<std::string>({});
+const std::vector<std::string> one_scalar_operand =
+    std::vector<std::string>({"%alpha"});
+const std::vector<std::string> one_algorithm_operand =
```

Comment:
It implies there are some fusion attributes that accept multiple algorithms. Is that true?

Reply:
Renamed to `algorithm_indicator` to avoid confusion.